modules/sharedIdSystem.js

```diff
@@ -54,19 +55,23 @@ function isIdSynced(configParams, storedId) {
     utils.logInfo('SharedId: Sync time is not configured or is not a number');
   }
   let syncTime = (!configParams || typeof configParams.syncTime !== 'number') ? DEFAULT_24_HOURS : configParams.syncTime;
+  if (syncTime < 0) {
```

Comment:
@jdwieland8282 reused syncTime parameter. Is this ok? Less than zero (e.g.`syncTime: -1`)  would mean sync is not required.

Reply:
Should this be <= 0 to match other config?